### PR TITLE
Add poezio

### PR DESCRIPTION
**Pull request by lsix**

### CODE REVIEW

**Comment by pSub** [comment]
As poezio is a cli application you might want to use `namePrefix = "";`.

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -2800,6 +2800,25 @@ let
...
+    disabled = (!isPy3k);
+
+    src = pkgs.fetchurl {
+      url = "https://pypi.python.org/packages/source/d/dnspython3/${name}.zip";
+      sha256 = "138wxj702vx6zni9g2y8dbgbpin95v6hk23rh2kwfr3q4130jqz9";
+    };
+
+    meta = {
+      description = "A DNS toolkit for Python 3.x";
+      homepage = "http://www.dnspython.org";
```
Comment (pSub) [inline pkgs/top-level/python-packages.nix]: You do not need quotes around urls.

File: pkgs/top-level/python-packages.nix [hunk 2]
```diff
@@ -16363,5 +16401,28 @@ let
...
+     sed -i -e "s/sleekxmpp==1.2.4/sleekxmpp==1.2.5/" setup.py
+   '';
+
+    src = pkgs.fetchurl {
+      url = "http://dev.louiz.org/attachments/download/52/poezio-0.8.1.tar.xz";
+      sha256 = "0n3phh3lc82609ssfvqvd4papvhykd1sf2bm88dggh2x4mypwjff";
+    };
+
+    meta = {
+      description = "Poezio is a free console XMPP client";
```
Comment (pSub) [inline pkgs/top-level/python-packages.nix]: Do not repeat the package name in the description. See http://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes

**Comment by lancelotsix** [comment]
Updated. Thanks for feedbacks.

**Comment by pSub** [comment]
Ah, one more thing. Do you want to add yourself as maintainer?

**Comment by lancelotsix** [comment]
Done.

I also included a small fix, let me know if the patch files should be placed somewhere else.

**Comment by lethalman** [comment]
Looks fine. Thanks and welcome.

**Pull request merged**